README.md

```diff
@@ -29,91 +66,28 @@ Two important things to do in order to properly handle deep links from Facebook
-
-6.) __Chrome seems to take me to Google Play all the time. Why?__
-
-Chrome is very picky about opening up the app directly. Chrome utilizes the intent system to try to open up the app, and fails back to the Play Store far too easily. Here are 3 things to verify:
-
-  1. Make sure the package name for the test build matches the package registered in the Branch settings and matches the package in Google play.
-
-  2. Make sure the URI scheme of the test build matches the Branch settings.
-
-  3. Verify that you've added the proper host 'open' in the Manifest - see [here](https://github.com/BranchMetrics/Branch-Android-SDK#register-an-activity-for-direct-deep-linking-optional-but-recommended)
-
-
-7.) __Why is the most recent version of Chrome (40+) is showing me a page not found error?__
-
-The Google Chrome team decided that it didn't want to try to open up the app if a user manually entered in a link into Chrome - see the history of this issue [here](https://code.google.com/p/chromium/issues/detail?id=331571). The scope of the bug applies to people who copy and paste a Branch link into Chrome, whereas anyone who clicks on a link in Chrome or something that opens the link in Chrome will properly redirect.
-
-We've filed an issue plus documented how you can file your own with the Chrome team to get this changed. Please see [this document](https://github.com/BranchMetrics/Branch-Android-SDK/blob/master/ChromeHelp.md) for more ways you can help!
+Have questions? Need troubleshooting assistance? See our [FAQs](https://dev.branch.io/references/android_sdk/#faq) for in depth answers.
 
 ## Installation
 
-Current compiled SDK footprint is *40kb*
-
-### Install library project
-
-Import the SDK as a Gradle dependency (for Android Studio):
-* Right click on the main module within your project (this is called 'app' by default).
-* Select **Open Module Settings**.
-* Within the **Dependencies** tab, click the **+** button at the bottom of the window and select **Library Dependency**.
-* Type *branch*, and hit the enter key to search Maven Central for the Branch SDK Library.
-* Select the latest *io.branch.sdk.android:library* item listed and accept the changes.
-
-See the [Android Quick Start Guide for more detail](https://github.com/BranchMetrics/Branch-Integration-Guides/blob/master/android-quick-start.md) and a screencasted walkthrough.
-
-Or download the JAR file from here:
-https://s3-us-west-1.amazonaws.com/branchhost/Branch-Android-SDK.zip
-
-The testbed project:
-https://s3-us-west-1.amazonaws.com/branchhost/Branch-Android-TestBed.zip
-
-Or just clone this project!
+For installation information, you can go to the [Android SDK Installation](https://dev.branch.io/references/android_sdk/#installation) section of our new [documentation portal] (https://dev.branch.io) or to the [Android Quick Start Guide](https://github.com/BranchMetrics/Branch-Integration-Guides/blob/master/android-quick-start.md).
```

Comment:
This should be the guide we point people to: https://dev.branch.io/recipes/quickstart_guide/android/